localization/english/wc_traits_l_english.yml

```diff
@@ -498,4 +498,8 @@
 
  trait_creature_nightborne:0 "Nightborne"
  trait_creature_nightborne_desc:0 "The Nightborne are a mystic race of elves originating from Suramar. Since the destruction of the Well of Eternity, the isolated Nightborne has evolved to become a new race thanks to the power of the Nightwell. Like other elves they require sustenance in the form of magic. Because of this the Nightborne employ the power of the Nightwell in the form of Arcwine to sustain themselves. The Nightborne remain the only elven race to maintain the legacy of the Kaldorei empire, sustaining the advancement of magic long after Azshara fell."
- trait_creature_nightborne_character_desc:0 "This character is a Nighborne elf. $trait_creature_nightborne_desc$"
\ No newline at end of file
+ trait_creature_nightborne_character_desc:0 "This character is a Nighborne elf. $trait_creature_nightborne_desc$"
+
+ trait_creature_drogbar:0 "Drogbar"
+ trait_creature_drogbar_desc:0 "The Drogbar are a race created by the Titans by modifying the Trogg. Formerly slaves to the black dragon Neltharion, the Drogbar live mostly in relative peace with their neighbors, the Highmountain Tauren. While some of the Drogbar continue to revere the black dragons, the vast majority worships a mix of the elements as well as Highmountain's gigantic worms, primarily among them Naraxas."
```

Comment:
I wouldn't mention what they worship because it may change during the gameplay.
Also, don't request and don't translate English to other languages before it's approved. They have to change their localization now.